src/se/edu/inclass/task/Event.py

```diff
@@ -0,0 +1,8 @@
+from task.Task import Task 
+
+class Event(Task):
```

Comment:
event would have date and start-time, end-time (in the fix branch) and mechanisms to populate + display those fields

Reply:
for fix branch - taken note